src/libappimage/CMakeLists.txt

```diff
@@ -1,16 +1,20 @@
 cmake_minimum_required(VERSION 3.2)
 
-set(libappimage_public_header ${PROJECT_SOURCE_DIR}/include/appimage/appimage.h)
+set(libappimage_public_headers_list
+        ${PROJECT_SOURCE_DIR}/include/appimage/appimage.h
+        ${PROJECT_SOURCE_DIR}/include/appimage/appimage_shared.h)
```

Comment:
`libappimage_shared`'s header should be installed in its own directory. This is breaking the separation.

Reply:
Actually, we agreed on putting all appimage related headers into the appimage dir. So I guess it right place.